app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -87,6 +88,9 @@ class AppointmentRepositoryAndroidTest {
   private val facility: Facility
     get() = testData.qaFacility()
 
+  private val userUuid: UUID
+    get() = testData.qaUserUuid()
```

Comment:
We cannot do this since `testData` is injected and not instantiated when the test is created. However, we can use a lazy delegate to make this more clear. What do you think?

Reply:
I've updated the test. Please check?

Reply:
This lgtm 👍